### PR TITLE
Remove node constraint

### PR DESCRIPTION
**Pull request by goszczynskip**
Removes node constraint and bumps canvas to "2.6.1".

Links to: #40

### CODE REVIEW

**Comment by goszczynskip** [comment]
"yarn test" run locally is passing (run on windows). Althoulgh I don't know if there are some false positives.

**Comment by Fil** [comment]
the test that fail is `yarn prepublishOnly`

**Comment by goszczynskip** [comment]
`yarn prepublishOnly` should pass now

**Comment by Fil** [comment]
ah yes, even simpler than https://github.com/d3/d3-geo-polygon/issues/40#issuecomment-652366638 :)
so maybe a solution is possible that would be compatible with both node-canvas@1 and node-canvas@2, using:
```
Canvas = require("canvas")
if (Canvas.Canvas) Canvas = Canvas.Canvas;
```

**Comment by goszczynskip** [comment]
Good idea! I've made changes as you requested.

File: test/render-world [hunk 1]
```diff
@@ -13,6 +13,9 @@ var fs = require("fs"),
     Canvas = require("canvas"),
     d3 = Object.assign({}, require("d3-geo"), require("../"));
 
+// canvas@2 compatibility check
+if(Canvas.Canvas) Canvas = Canvas.Canvas;
```
Comment (mbostock) [inline test/render-world:17]:
```suggestion
if (Canvas.Canvas) Canvas = Canvas.Canvas;
```

**Comment by Fil** [comment]
thank you!

**Pull request merged**